bool new_pcd_wanted
---
bool result
